index.html

```diff
@@ -1318,19 +1318,17 @@ <h4 id="h-duration-change-algorithm" resource="#h-duration-change-algorithm"><sp
           <p>Follow these steps when <code><a href="#widl-MediaSource-duration">duration</a></code> needs to change to a <var>new duration</var>.</p>
           <ol>
             <li>If the current value of <code><a href="#widl-MediaSource-duration">duration</a></code> is equal to <var>new duration</var>, then return.</li>
-            <li>Set <var>old duration</var> to the current value of <code><a href="#widl-MediaSource-duration">duration</a></code>.
-            </li><li>Update <code><a href="#widl-MediaSource-duration">duration</a></code> to <var>new duration</var>.</li>
-            <li>If the <var>new duration</var> is less than <var>old duration</var>, then run the
-              <a href="#sourcebuffer-range-removal">range removal</a> algorithm with <var>new duration</var> and
-              <var>old duration</var> as the start and end of the removal range.
-
-              <div class="note"><div class="note-title marker" aria-level="5" role="heading" id="h-note15"><span>Note</span></div><p class="">This preserves audio frames and text cues that start before and end after the <code><a href="#widl-MediaSource-duration">duration</a></code>.</p></div></li>
-            <li>If a user agent is unable to partially render audio frames or text cues that start before and end after the <code><a href="#widl-MediaSource-duration">duration</a></code>, then run the following steps:
+			<li>If <var>new duration</var> is less than the highest starting <a href="#presentation-timestamp">presentation timestamp</a> reported by the <code><a href="#widl-SourceBuffer-buffered">buffered</a></code> attribute across all <a href="#idl-def-SourceBuffer" class="idlType"><code>SourceBuffer</code></a> 
```

Comment:
For a little more clarity on why this current phrasing could lead to implicit removal of the entire presentation interval of coded frame(s), consider the example single, un-muxed SourceBuffer whose .buffered reports one range beginning at time 0, but having 100 frames at 30ms intervals. If the app attempted to reduce the duration to 0, this phrasing would allow that without exception, but would implicitly remove all but perhaps the very first coded frame. This violates the intent of fixing these issues, which is to require explicit removal be done for any frame whose entire presentation interval lies after the new duration.